cmake/ArborXConfig.cmake.in

```diff
@@ -5,7 +5,6 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR} ${CMAKE_MODULE_PATH})
 include(CMakeFindDependencyMacro)
 find_dependency(Kokkos)
 
-include("${CMAKE_CURRENT_LIST_DIR}/ArborXSettings.cmake")
 if(ARBORX_ENABLE_MPI)
```

Comment:
Where do you get that variable from in downstream projects?

Reply:
Do we need that as a `CMake` variable? Otherwise, it is defined in `/ArborX_Config.hpp.in` resp.`ArborX_Config.hpp`.

Reply:
Look at the next line.